/**:
  ros__parameters:
    control_command: "raw"
    recordreplay_command_enabled: false
    # Which index in the joy message corresponds to which axis
    axes:
      # In our version of the joystick driver, axes typically go from -1.0 to 1.0
      left_cmd: 1
      right_cmd: 3
    # Scale and offset of axes: val = (raw * scale) + offset
    axis_scale:
      left_cmd: 100.0
      right_cmd: 100.0
      # curvature:
      # acceleration:
    axis_offset:
      left_cmd: 0.0
      right_cmd: 0.0
    # Buttons are not yet implemented
    # Index for which button is which
    buttons:
      recordreplay_start_record : 7 # start button
      recordreplay_start_replay : 8 # logo button
      recordreplay_stop : 6 # back button
